components/org.wso2.carbon.security.caas/src/main/java/org/wso2/carbon/security/caas/internal/CarbonSecurityComponent.java

```diff
@@ -284,36 +283,35 @@ private void initAuthorizationConfigs(BundleContext bundleContext) {
     private void setDefaultPermissions(BundleContext context) {
 
         PermissionAdmin permissionAdmin = getPermissionAdmin(context);
+
         if (permissionAdmin == null) {
             return;
         }
 
-        DefaultPermissionInfoCollection permissionInfoCollection = SecurityConfigBuilder
-                .buildDefaultPermissionInfoCollection();
-        if (Collections.EMPTY_SET.equals(permissionInfoCollection.getPermissions())) {
-            throw new RuntimeException("Default permission info collection can't be empty.");
+        PermissionConfigFile permissionConfigFile = null;
+        try {
+            permissionConfigFile = PermissionConfigBuilder.buildPermissionConfig();
+        } catch (PermissionConfigException e) {
+            log.error("Error occurred in building the permission configuration - ", e);
```

Comment:
Remove the '-' at the end of the exception message. The message you give will be printed in a single line followed by the stack trace.